### ISSUE TITLE
Dataset Files Tab: Tabular data file category missing from popup.

### ISSUE THREAD

**Issue by kcondon** [IQSS/dataverse#2666]
From the dataset page, files tab, if you edit tags, the popup does not include the tabular file categories if at least on tabular file was selected.

Leonid said to please wait until he has finished his checkins/ merges before checking in a fix for this.

**Comment by landreev** [comment]
Hmmm. I have a bad feeling that that was merely a side effect of the annoying "wrong file" bug being back. 
I.e., you weren't seeing the tabular categories, simply because the page was actually trying to get them from a wrong file. That wasn't tabular at all...
Same reason you get a wrong file when you try to download a file. 

Steve, Kevin saw this while looking at a build before I merged my code into the branch; of course I have merged it since then... so I'm not even sure how to proceed... Maybe it would actually help to look into trying to reproduce this *before* you pull latest changes?? - These can definitely be entirely separate issues: the missing data categories; and the evil "wrong file" bug, that I'm seeing on my build now. I may have introduced the latter.

**Comment by sekmiller** [comment]
I’ll take a look at 2666. I’m guessing it’s a side effect of updating the selected files lists on button press instead of file select.


On Oct 21, 2015, at 11:47 PM, landreev <notifications@github.com<mailto:notifications@github.com>> wrote:


Hmmm. I have a bad feeling that that was merely a side effect of the annoying "wrong file" bug being back.
I.e., you weren't seeing the tabular categories, simply because the page was actually trying to get them from a wrong file. That wasn't tabular at all...
Same reason you get a wrong file when you try to download a file.

Steve, Kevin saw this while looking at a build before I merged my code into the branch; of course I have merged it since then... so I'm not even sure how to proceed... Maybe it would actually help to look into trying to reproduce this before you pull latest changes?? - These can definitely be entirely separate issues: the missing data categories; and the evil "wrong file" bug, that I'm seeing on my build now. I may have introduced the latter.
